### PR TITLE
Handle edge case in assert

### PR DESCRIPTION
**Pull request by alexkeating**

### CODE REVIEW

File: src/test/BinaryEligibilityOracleEarningPowerCalculatorTestBase.sol [hunk 1]
```diff
@@ -42,30 +42,12 @@ abstract contract BinaryEligibilityOracleEarningPowerCalculatorTestBase is Stake
     vm.stopPrank();
   }
 
-  /// @notice A test helper that wraps calling the `stake` function on the underlying Staker
```
Comment (funkyenough) [inline src/test/BinaryEligibilityOracleEarningPowerCalculatorTestBase.sol:32]: (Arbitrary Comment related to Line 36)

Here's a nit: with the refactor, `_setDelegateeScoreAboveThreshold` is only used in `StakedBinaryEligibilityOracleEarningPowerCalculatorTestBase`. It is probably cleaner to move the function there, and potentially just merge the logic into `_stakeBelowThreshold` if it not going to be used anywhere else.
